framework/manager.go

```diff
@@ -108,7 +108,8 @@ func (m *PluginManager) GeneratePolicy(ctx context.Context, pluginSet map[string
 	for providerId, policyPlugin := range pluginSet {
 		componentTitle, ok := m.pluginIdMap[providerId]
 		if !ok {
-			return fmt.Errorf("missing title for provider %s", providerId)
+			m.log.Info(fmt.Sprintf("skipping %s provider: missing validation component", providerId))
```

Comment:
Looks good. Can this be update be a `Warning` instead of `Info`?

Reply:
Just updated. Thanks @jpower432